Coalesce About tab to splash screeen. Create support and contact subsections (#143)
ACCESS-Hive/ACCESS-Hive: docs/index.md

# ACCESS-Hive

![Network Logo](assets/ACCESSLogo(RGB).png){align=right width=40%}
ACCESS-Hive is a portal to all documentation relevant to the Australian Community Climate and Earth System Simulator, ACCESS, and the wider ACCESS community. ACCESS-Hive provides links to documentation hosted on other sites. It is not intended to host the documentation itself.

The ACCESS community has indicated a need for high quality documentation. ACCESS-Hive is the first step in this process by improving accessibility to existing documentation. There is immense value to the broader community to have a single location to find high quality, relevant documentation. ACCESS-Hive will also be a valuable information source when building ACCESS-NRI supported documentation.

ACCESS-Hive is curated to ensure it is relevant and of reasonable quality, and all links have an annotated summary to help you find what you need. Annotations also makes the in-built live search function extremely effective.

## Support

There is a system of tags to identify who supports the linked documentation or software, and the level of support you can expect:

> {{supp}} Supported by ACCESS-NRI

Reviewer: ```suggestion
<div align="center"> {{supp}} Supported by ACCESS-NRI </div>
```
I'm trying to highlight this a bit more. Not sure that's the right solution though. But I think it would look better if it was tabulated somehow rather than align="left"

> {{recom}} Recommended by ACCESS-NRI

Reviewer: ```suggestion
<div align="center"> {{recom}} Recommended by ACCESS-NRI </div>
```

> {{comm}} Community contributed

Reviewer: ```suggestion
<div align="center"> {{comm}} Community contributed </div>
```

See the [support page](about/support.md) for details about the support levels: what is supported, by who, and how to access help.

## Contributions

[How to Contribute][HCG]{ .md-button}

> ACCESS-Hive is a community supported site, as such contributions to the ACCESS-Hive site are **encouraged by any member of the community**. Please refer to the [How to contribute guide][HCG] to learn how you can help the ACCESS community build a documentation database useful to everyone.

Reviewer: ```suggestion
ACCESS-Hive is a community supported site, as such contributions to the ACCESS-Hive site are **encouraged by any member of the community**. Please refer to the [contribution guidelines][HCG] to learn how you can help the ACCESS community build a documentation database useful to everyone.
```
You haven't change that but I think it sounds better.
